src/components/Avatar/Avatar.vue

```diff
@@ -260,7 +262,7 @@ export default {
 			if (this.isMenuLoaded) {
 				return this.menu.length > 0
 			}
-			return !(this.user === OC.getCurrentUser().uid || this.userDoesNotExist || this.url)
+			return !(this.user === getCurrentUser().uid || this.userDoesNotExist || this.url)
```

Comment:
the replacement works different as it may return null when the user is not known. so it should be `getCurrentUser()?.uid`

Reply:
Wasn't aware of this. I will provide a PR later.

Reply:
Fix is in #1039.